multiscale_phate/multiscale_phate.py

```diff
@@ -2,28 +2,43 @@
 
 
 class Multiscale_PHATE(object):
-    """Short summary.
+    """Multscale PHATE operator which performs dimensionality reduction and clustering across granularities.
 
     Parameters
     ----------
-    scale : type
-        Description of parameter `scale`.
-    landmarks : type
-        Description of parameter `landmarks`.
-    partitions : type
-        Description of parameter `partitions`.
-    granularity : type
-        Description of parameter `granularity`.
-    n_pca : type
-        Description of parameter `n_pca`.
-    decay : type
-        Description of parameter `decay`.
-    gamma : type
-        Description of parameter `gamma`.
-    knn : type
-        Description of parameter `knn`.
-    n_jobs : type
-        Description of parameter `n_jobs`.
+    scale : float, default: 1.025
+        speed at which epsilon increases from iteration to
+        iteration
+    landmarks : int, default: 2000
+        number of landmarks to compute diffusion potential
+        coordinates on
+    partitions : int, default: None
+        number of partitions to split data into in initial
+        coarse graining. Only applies ot large datasets
+    granularity : float, default: .1
+        Fraction of silverman bandwidth to set initial
+         kernel bandwidth to
+    n_pca : int, default: 100
+        Number of principal components to use for calculating
+        neighborhoods. For extremely large datasets, using
+        n_pca < 20 allows neighborhoods to be calculated in
+        roughly log(n_samples) time.
+    decay : int, default: 40
+        sets decay rate of kernel tails in diffusion potential
+        calculation. If None, alpha decaying kernel is not used
+    gamma : float, optional, default: 1
+        Informational distance constant between -1 and 1.
+        `gamma=1` gives the diffusion potential log potential,
+        while `gamma=0` gives a square root potential.
+    knn : int, optional, default: 5
+        number of nearest neighbors on which to build kernel for
+        diffusion potential calculation.
+    n_jobs : integer, optional, default: 1
+        The number of jobs to use for the computation.
+        If -1 all CPUs are used. If 1 is given, no parallel computing code is
+        used at all, which is useful for debugging.
+        For n_jobs below -1, (n_cpus + 1 + n_jobs) are used. Thus for
+        n_jobs = -2, all CPUs but one are used
 
     Attributes
```

Comment:
Attributes list should document what all the following are:
```
        self.NxTs = None
        self.Xs = None
        self.Ks = None
        self.merges = None
        self.Ps = None
        self.diff_op = None
        self.data_pca = None
        self.pca_op = None
        self.partition_clusters = None
        self.dp_pca = None
        self.epsilon = None
        self.merge_threshold = None
        self.gradient = None
        self.levels = None
```

Reply:
You should describe these -- they are the primary attributes of your estimator. I have no idea what NxTs means, for example